src/components/Settings/Settings.css

```diff
@@ -89,7 +89,7 @@ body{
   padding:20px 20px 0px 30px;
   width:598.5px;
   min-height:500px;
-  height: fit-content;
+  height: 300px;
```

Comment:
This would only be a temporary fix, one thing you could do is take the nav and the right section and keep them in a single div that would ensure their heights are same. Also follow - https://chris.beams.io/posts/git-commit/

Reply:
@anshumanv Done the changes.